Movie/MovieTests/MainScreenPresenterTest.swift

```diff
@@ -0,0 +1,96 @@
+// MainScreenPresenterTest.swift
+// Copyright © RoadMap. All rights reserved.
+
+@testable import Movie
+import XCTest
+
+class MockView: MainViewProtocol {
+    func successToFetchMovies() {}
+    func failToFetchMovies(withError error: Error) {}
+}
+
+class MockMovieAPIService: MovieAPIServiceProtocol {
+    var movies: [Movie]!
+
+    init() {}
+
+    convenience init(movies: [Movie]?) {
+        self.init()
+        self.movies = movies
+    }
+
+    func fetchMovies(withURLString urlString: String, completion: @escaping (Result<[Movie]?, Error>) -> Void) {
+        if let movies = movies {
+            completion(.success(movies))
+        } else {
+            let error = NSError(domain: "", code: 0, userInfo: nil)
+            completion(.failure(error))
+        }
+    }
+
+    func fetchImage(forMovie movie: Movie?, completion: @escaping (UIImage) -> Void) {}
+}
+
+class MainScreenPresenterTest: XCTestCase {
```

Comment:
```suggestion
class MainScreenPresenterTest: XCTestCase {
```
не надо же было пока в этой дз это в пятницу